SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -126,22 +127,31 @@ void USpatialGameInstance::StartGameInstance()
 
 bool USpatialGameInstance::ProcessConsoleExec(const TCHAR* Cmd, FOutputDevice& Ar, UObject* Executor)
 {
-	if (!Super::ProcessConsoleExec(Cmd, Ar, Executor))
+	if (Super::ProcessConsoleExec(Cmd, Ar, Executor))
 	{
-		if (GetWorld() == nullptr)
-		{
-			return false;
-		}
+		return true;
+	}
 
-		USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(GetWorld()->GetNetDriver());
-		if (NetDriver == nullptr || NetDriver->SpatialMetrics == nullptr)
+	if (const UWorld* World = GetWorld())
+	{
+		if (const USpatialNetDriver* NetDriver = Cast<USpatialNetDriver>(World->GetNetDriver()))
 		{
-			return false;
-		}
+			if (NetDriver->SpatialMetrics && NetDriver->SpatialMetrics->ProcessConsoleExec(Cmd, Ar, Executor))
+			{
+				return true;
+			}
 
-		return NetDriver->SpatialMetrics->ProcessConsoleExec(Cmd, Ar, Executor);
+			if (GetDefault<USpatialGDKSettings>()->bEnableMetricsDisplay)
```

Comment:
I think it's worth removing this check, if the display object exists, it's available for processing.